src/core/reactor_backend.cc

```diff
@@ -178,7 +178,7 @@ aio_storage_context::submit_work() {
     bool did_work = false;
 
     _submission_queue.resize(0);
-    size_t to_submit = _r._io_sink.drain([this] (internal::io_request& req, io_completion* desc) -> bool {
+    size_t to_submit = _r._io_sink.drain([this] (const internal::io_request& req, io_completion* desc) -> bool {
```

Comment:
The `io_sink::drain()` should be patched to manipulate const io_requests

Reply:
there is a dedicated commit to implement the changes in `io_sink`. see 7eb4ec70159e2e60f8d9be0ed40636af85fa3afc

Reply:
Indeed